src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3833,18 +3844,13 @@ namespace HVACUnitaryBypassVAV {
             DataLoopNode::Node(MixerInletAirNode).MassFlowRate = AverageUnitMassFlow;
             DataLoopNode::Node(MixerOutsideAirNode).MassFlowRate = AverageOAMassFlow;
             DataLoopNode::Node(MixerReliefAirNode).MassFlowRate = AverageOAMassFlow;
-            if (FirstHVACIteration) {
-                OnOffAirFlowRatio = 1.0;
-                BypassDuctFlowFraction = 0.0;
-            } else {
-                OnOffAirFlowRatio = 1.0;
-                auto &cbVAVBoxOut(CBVAV(CBVAVNum).CBVAVBoxOutletNode);
-                Real64 boxOutletNodeFlow = 0.0;
-                for (int i = 1; i <= CBVAV(CBVAVNum).NumControlledZones; ++i) {
-                    boxOutletNodeFlow += DataLoopNode::Node(cbVAVBoxOut(i)).MassFlowRate;
-                }
-                BypassDuctFlowFraction = max(0.0, 1.0 - (boxOutletNodeFlow / AverageUnitMassFlow));
+            OnOffAirFlowRatio = 1.0;
```

Comment:
So here the BypassDuctFlowFration was set = 0 and the model needed to iterate enough to get to the correct bypass fraction. I figured to just set it based on the TU box flow rate and always know the correct fraction.